mapbox/dr-ui
# [patch] Use babel.config.js and updates build-package.js to output directories

This patches the last release, 0.8.0 #117 . When `npm run build` the components were being packaged as files instead of directories, which broke several components that reference other dr-ui components. I updated the babel config and the build script (thanks to mr-ui prior art) and now when packaged, the components are saved as directories once again.

To test this PR, you can assert that you find:
* No errors on `npm start`
* No errors on `npm run build` and `pkg/` directory shows a folder for each component
* No errors on `npm test`

I linked this local version to /ios locally to assert that the packaged dr-ui release does not throw errors. We should definitely do this step if we change anything to the build script in the future.
